### PR TITLE
PMM-11426 run tests on pr

### PR DESCRIPTION
**Pull request by yurkovychv**
https://jira.percona.com/browse/PMM-11426

- https://github.com/percona/pmm-ui-tests/pull/494

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3021-8b694a6
client docker - perconalab/pmm-client-fb:PR-3021-8b694a6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3021-8b694a6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3021-8b694a6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3021-8b694a6.tar.gz

File: .github/workflows/ui-tests.yml [hunk 1]
```diff
@@ -141,7 +141,7 @@ jobs:
           path: tests/output
 
       - name: Create status check
-        if: ${{ always() }}
+        if: ${{ github.event_name != 'pull_request' && always() }}
```
Comment (BupycHuk) [inline .github/workflows/ui-tests.yml:144]: this line makes no sense, if we need to create a status check for everything except pull requests then let's leave only the first part, if we need to create a status check always then let's keep the old logic.

Comment (yurkovychv) [reply .github/workflows/ui-tests.yml:144]: `always` means that this step will execute even if the workflow execution failed in the middle. Basically to always send status

Comment (BupycHuk) [reply .github/workflows/ui-tests.yml:144]: ah, ok

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**